services/alertmanager/alertmanager.go

```diff
@@ -352,7 +447,15 @@ func (svc *Service) populateConfig(cfg *alertmanager.Config) error {
 		return nil
 	})
 	if e != nil {
-		return errors.Errorf("Failed to fetch items from database: %s", e)
+		return errors.Errorf("failed to fetch items from database: %s", e)
```

Comment:
minor: I was told to use `%v` instead of `%s` for error

Reply:
thanks, changed everywhere